examples/cluster/plot_adjusted_for_chance_measures.py

```diff
@@ -67,8 +68,12 @@ def ami_score(U, V):
     metrics.mutual_info_score,
 ]
```

Comment:
I think we should add normalized mutual information and non-adjusted rand score.

Also we should probably reorder this list to make the legend of the plots easier to read:


```python
score_funcs = [
    metrics.v_measure,
    metrics.rand_score,
    metrics.adjuster_rand_score,
    metrics.mutual_info_score,
    metrics.normalized_info_score,
    metrics.adjusted_mutual_info_score,
]
```